frontend/src/app/modules/bim/ifc_models/toolbar/view-toggle/bim-view-toggle-dropdown.directive.ts

```diff
@@ -68,35 +70,38 @@ export class BimViewToggleDropdownDirective extends OpContextMenuTrigger {
   private buildItems() {
     const current = this.bimView.current;
     const viewRoute = this.state.current.data.viewRoute;
+    let items = this.viewerBridgeService.shouldShowViewer ?
+                  [bimViewerViewIdentifier, bimListViewIdentifier, bimSplitViewIdentifier] :
+                  [bimListViewIdentifier];
 
-    this.items = [bimViewerViewIdentifier, bimListViewIdentifier, bimSplitViewIdentifier]
-      .map(key => {
-        return {
-          hidden: key === current,
-          linkText: this.bimView.text[key],
-          icon: this.bimView.icon[key],
-          onClick: () => {
-            // Close filter section
-            if (this.wpFiltersService.visible) {
-              this.wpFiltersService.toggleVisibility();
-            }
+    this.items = items
+                  .map(key => {
+                      return {
+                      hidden: key === current,
+                      linkText: this.bimView.text[key],
+                      icon: this.bimView.icon[key],
+                      onClick: () => {
+                          // Close filter section
+                          if (this.wpFiltersService.visible) {
+                          this.wpFiltersService.toggleVisibility();
+                          }
 
-            switch (key) {
-              case bimListViewIdentifier:
-                this.state.go('bim.partitioned.list');
-                break;
-              case bimViewerViewIdentifier:
-                this.state.go('bim.partitioned.model');
-                break;
-              case bimSplitViewIdentifier:
-                this.state.go('bim.partitioned.split');
-                break;
-            }
+                          switch (key) {
+                          case bimListViewIdentifier:
+                              this.state.go('bim.partitioned.list', {cards: true});
```

Comment:
@aleixsuau We usually add a space after an opening curly brace and before an closing one. So `{ cards: true }` instead of `{cards: true}`.